src/test/java/org/snomed/snowstorm/core/data/services/BranchMergeServiceTest.java

```diff
@@ -3630,6 +3630,405 @@ void assertReleaseProperties(
 		}
 	}
 
+	@Test
+	void testRebaseDoesNotResultInExtensionStealingAssociationMemberWhenNoConflict() throws ServiceException, InterruptedException {
```

Comment:
This test shows that when there is no conflict, inactive historical associations do not switch module.